### PR TITLE
Add more contact info

### PR DESCRIPTION
**Pull request by matthiasbeyer**
More contact information in the sidebar.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@fkemmer please review.
@TheNeikos you are welcome to review as well.

File: layouts/default.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
                 <li><%= link_to "Tags", '/tags/' %></li>
                 <li><%= link_to "Atom", '/atom/' %></li>
                 <li><%= link_to "XML Sitemap", '/sitemap/' %></li>
+                <li><a href='mailto:orga[at]unfug(dot)org'>Mail us</a></li>
```
Comment (TheNeikos) [inline layouts/default.html:24]: Interesting mailto address, put it rather in clear text.

File: layouts/default.html [hunk 2]
```diff
@@ -21,6 +21,9 @@
...
+                <li><a href='irc://irc.ghb.fh-furtwangen.de:6667/#unfug'>IRC</a></li>
```
Comment (TheNeikos) [inline layouts/default.html:25]: For those that don't support the protocol will have to check the source or hope their browser shows it on hover, please put it in clear text as well.

Comment (matthiasbeyer) [inline layouts/default.html:25]: You'd suggest to write `#unfug@irc.ghb-hs-furtwangen.de` as link title?

Comment (TheNeikos) [inline layouts/default.html:25]: Just #unfug on irc.ghb.hs-furtwangen.de

Comment (matthiasbeyer) [inline layouts/default.html:25]: That's pretty much text for a sidebar,...

File: layouts/default.html (hunk 1 above)
Comment (matthiasbeyer) [inline layouts/default.html:24]: The point is that automatic page crawlers are not able to send spam, but a human can easily fix this. I'd like to keep it this way.

What do you think, @fkemmer ?

**Pull request merged**